hyrisecockpit/frontend/src/components/metrics/Access.vue

```diff
@@ -28,13 +28,14 @@
         :id="'access-select'"
         v-model="selectedItem"
         class="select"
-        :items="selectionItems"
-        chips
+        :items="selectedItems"
```

Comment:
```suggestion
         :items="selectionItems"
```
Sorry, I just made a typo in my last review, this is the correct value